content/ko/docs/reference/scheduling/config.md

```diff
@@ -0,0 +1,253 @@
+- `VolumeBinding`: 노드에 요청된 {{< glossary_tooltip text="볼륨" term_id="volume" >}}이 있는지
+  또는 바인딩할 수 있는지 확인한다.
+  익스텐션 포인트: `PreFilter`, `Filter`, `Reserve`, `PreBind`.
+- `VolumeRestrictions`: 노드에 마운트된 볼륨이 볼륨 제공자에 특정한
+  제한 사항을 충족하는지 확인한다.
+  익스텐션 포인트: `Filter`.
+- `VolumeZone`: 요청된 볼륨이 가질 수 있는 영역 요구 사항을 충족하는지
+  확인한다.
+  익스텐션 포인트: `Filter`.
+- `NodeVolumeLimits`: 노드에 대해 CSI 볼륨 제한을 충족할 수 있는지
+  확인한다.
+  익스텐션 포인트: `Filter`.
+- `EBSLimits`: 노드에 대해 AWS EBS 볼륨 제한을 충족할 수 있는지 확인한다.
+  익스텐션 포인트: `Filter`.
+- `GCEPDLimits`: 노드에 대해 GCP-PD 볼륨 제한을 충족할 수 있는지 확인한다.
+  익스텐션 포인트: `Filter`.
+- `AzureDiskLimits`: 노드에 대해 Azure 디스크 볼륨 제한을 충족할 수 있는지
+  확인한다.
+  익스텐션 포인트: `Filter`.
+- `InterPodAffinity`: [파드 간 어피니티 및 안티-어피니티](/ko/docs/concepts/scheduling-eviction/assign-pod-node/#파드간-어피니티와-안티-어피니티)를
+  구현한다.
+  익스텐션 포인트: `PreFilter`, `Filter`, `PreScore`, `Score`.
+- `PrioritySort`: 기본 우선 순위 기반 정렬을 제공한다.
+  익스텐션 포인트: `QueueSort`.
+- `DefaultBinder`: 기본 바인딩 메커니즘을 제공한다.
+  익스텐션 포인트: `Bind`.
+- `DefaultPreemption`: 기본 선점 메커니즘을 제공한다.
+  익스텐션 포인트: `PostFilter`.
+
+기본적으로 활성화되지 않은 컴포넌트 구성 API를 통해
+다음의 플러그인을 활성화할 수도 있다.
+
+- `NodeResourcesMostAllocated`: 리소스 할당이 많은 노드를
+  선호한다.
+  익스텐션 포인트: `Score`.
+- `RequestedToCapacityRatio`: 할당된 리소스의 구성된 기능에 따라 노드를
+  선호한다.
+  익스텐션 포인트: `Score`.
+- `NodeResourceLimits`: 파드 리소스 제한을 충족하는 노드를 선호한다.
+  익스텐션 포인트: `PreScore`, `Score`.
+- `CinderVolume`: 노드에 대해 OpenStack Cinder 볼륨 제한을 충족할 수 있는지
+  확인한다.
+  익스텐션 포인트: `Filter`.
+- `NodeLabel`: Filters and / or scores a node according to configured
+  {{< glossary_tooltip text="label(s)" term_id="label" >}}.
+  익스텐션 포인트: `Filter`, `Score`.
+- `ServiceAffinity`: {{< glossary_tooltip text="서비스" term_id="service" >}}에
+  속한 파드가 구성된 레이블로 정의된 노드 집합에 맞는지
+  확인한다. 이 플러그인은 또한 서비스에 속한 파드를 노드 간에
+  분산하는 것을 선호한다.
+  익스텐션 포인트: `PreFilter`, `Filter`, `Score`.
+
+### 여러 프로파일
+
+둘 이상의 프로파일을 실행하도록 `kube-scheduler` 를 구성할 수 있다.
+각 프로파일에는 연관된 스케줄러 이름이 있으며 [익스텐션 포인트](#익스텐션-포인트)에 구성된
+다른 플러그인 세트를 가질 수 있다.
+
+다음의 샘플 구성을 사용하면, 스케줄러는 기본 플러그인이 있는
+프로파일과 모든 스코어링 플러그인이 비활성화된 프로파일의 두 가지 프로파일로
+실행된다.
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta1
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: default-scheduler
+  - schedulerName: no-scoring-scheduler
+    plugins:
+      preScore:
+        disabled:
+        - name: '*'
+      score:
+        disabled:
+        - name: '*'
+```
+
+특정 프로파일에 따라 스케줄하려는 파드는
+`.spec.schedulerName` 에 해당 스케줄러 이름을 포함할 수 있다.
+
+기본적으로, 스케줄러 이름 `default-scheduler` 를 가진 하나의 프로파일이 생성된다.
+이 프로파일에는 위에서 설명한 기본 플러그인이 포함되어 있다. 둘 이상의
+프로파일을 선언할 때, 각각에 대한 고유한 스케줄러 이름이 필요하다.
+
+파드가 스케줄러 이름을 지정하지 않으면, kube-apiserver는 이를 `default-scheduler` 로
+설정한다. 따라서, 해당 파드를 스케줄하려면 이 스케줄러 이름을 가진 프로파일이
+있어야 한다.
+
+{{< note >}}
+파드의 스케줄링 이벤트에는 ReportingController로 `.spec.schedulerName` 이 있다.
+리더 선출을 위한 이벤트는 목록에서 첫 번째 프로파일의 스케줄러 이름을
+사용한다.
+{{< /note >}}
+
+{{< note >}}
+모든 프로파일은 QueueSort 익스텐션 포인트에서 동일한 플러그인을 사용해야 하며
+동일한 구성 파라미터(해당하는 경우)를 가져야 한다. 이것은 스케줄러에 보류 중인 파드 대기열이
+하나만 있기 때문이다.
```

Comment:
```suggestion
동일한 구성 파라미터(해당하는 경우)를 가져야 한다. 그 이유는 스케줄러가 보류 중 상태인 파드 대기열을
단 하나만 가질 수 있기 때문이다.
```
는 어떨까요?

This is because the scheduler only has one pending pods queue.

Reply:
반영하겠습니다.